package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "list-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "prisma generate && prisma migrate deploy && next build",
```

Comment:
Running `prisma generate` and `prisma migrate deploy` keeps the prisma client and database schema up to date. 

In a production application with a more complex dev cycle, you may not want to include `prisma migrate deploy` as part of the build process, but I find it works well when just starting small projects.